handlers/analytics/log_processor.go

```diff
@@ -166,6 +169,14 @@ func (p *LogProcessor) sendEvents() {
 
 	err := p.writer.WriteMessages(context.Background(), msgs...)
```

Comment:
Added in 0a9fee84c7ef7adb07740618754203059cc7ae5d

FYI: I also switched from census lib to promhttp, because I realize we used census only for mapic.